addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -575,6 +575,7 @@ def _prepare_info(self, allowed_picking_states):
             "result_package_names": pickings.get_result_packages_names(),
             "u_original_name": self.u_original_name,
             "has_done_pickings": has_done_pickings,
+            "u_last_reserved_pallet_name": pickings.get_result_packages_names() and self.u_last_reserved_pallet_name or False,
```

Comment:
Since this is going to be JSON'd, do we want to use `or None` so that we emit a JSON `null`?